### PR TITLE
add tests and linter

### PR DESCRIPTION
**Pull request by earthhuman41**

### CODE REVIEW

File: src/bankAccount.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+const BankAccount = require('./bankAccount');
+
+test('Should return account info', () => {
+  const account = new BankAccount('Dmytro', '26.11.1994', 1000);
+
+  expect(account.getInfo())
+    .toBe(`Name: Dmytro, Age: 26, Amount: 1000$`);
+});
+
```
Comment (mgrinko) [inline src/bankAccount.test.js]: Please add a separate test for initial history (before any operarion)

File: src/bankAccount.test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  expect(account.getAccountHistory()[0])
+    .toBe('Initial: 1700');
+
+  expect(account.getAccountHistory()[1])
+    .toBe('salary: 2000');
+
+  expect(account.getAccountHistory()[2])
+    .toBe('new phone: -500');
+});
```
Comment (mgrinko) [inline src/bankAccount.test.js]: 1) please create separate tests for adding money and taking
2) Please control the getAccountHistory() result type and length

Comment (YuriiHoliuk) [reply src/bankAccount.test.js]: agree with Misha, write small tests, for 1 method per tests, then you can write more complex integration tests

File: src/bankAccount.test.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const BankAccount = require('./bankAccount');
+
+test('Should return account info', () => {
+  const account = new BankAccount('Dmytro', '26.11.1994', 1000);
+
+  expect(account.getInfo())
+    .toBe(`Name: Dmytro, Age: 26, Amount: 1000$`);
```
Comment (YuriiHoliuk) [inline src/bankAccount.test.js]: move Dmytro and 1000 to variables and use here or do not use template string

File: src/bankAccount.test.js (hunk 1 above)
Comment (YuriiHoliuk) [reply src/bankAccount.test.js]: Also, will be great to add test for correct constructor
```
const account = new BankAccount('Dmytro', '26.11.1994', 1000);

```
check account is object and and check if account has correct fields

File: src/bankAccount.test.js (hunk 3 above)
Comment (mgrinko) [reply src/bankAccount.test.js]: @YuriiHoliuk It is a test to it is ok to use hardcoded values

Comment (mgrinko) [reply src/bankAccount.test.js]: But the test depends on current date 
So it will be broken at some moment

File: src/bankAccount.test.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    .not.toBe(undefined);
+
+  expect(account.getInfo)
+    .not.toBe(undefined);
+
+  expect(account.addMoney)
+    .not.toBe(undefined);
+
+  expect(account.withdrawMoney)
+    .not.toBe(undefined);
```
Comment (mgrinko) [inline src/bankAccount.test.js]: You can instead check if it is a function

**Pull request merged**